dev-java/jakarta-oro/jakarta-oro-2.0.8-r5.ebuild

```diff
@@ -11,22 +11,22 @@ MY_J=${PN%%-*}
 MY_O=${PN##*-}
 
 DESCRIPTION="A set of text-processing Java classes"
-HOMEPAGE="http://jakarta.apache.org/oro/index.html"
-SRC_URI="http://archive.apache.org/dist/${MY_J}/${MY_O}/${P}.tar.gz"
+HOMEPAGE="https://jakarta.apache.org/oro/"
+SRC_URI="https://archive.apache.org/dist/${MY_J}/${MY_O}/${P}.tar.gz"
 
 LICENSE="Apache-1.1"
 SLOT="2.0"
 KEYWORDS="~amd64 ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
-IUSE=""
 
-DEPEND=">=virtual/jdk-1.6"
-RDEPEND=">=virtual/jre-1.6"
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
 
 S="${WORKDIR}/${P}"
 
 JAVA_SRC_DIR="src"
 
 java_prepare() {
```

Comment:
should be `src_prepare` when using eapi7

Reply:
Ahh, sorry ...